### PR TITLE
Update game

### PR DESCRIPTION
**Pull request by TheBestChelik**
Pull request

### CODE REVIEW

**Comment by jusikXL** [comment]
Good job, man. But wait for one thing to be changed!

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Comment (jusikXL) [inline index.html:50]: Bad practice.

**Review by jusikXL — CHANGES_REQUESTED** [review]

File: index.html [hunk 2]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again, good practice 🧑‍💻 🤖!</h2>
```
Comment (jusikXL) [inline index.html:50]: Good fix, man

File: README.md [hunk 3]
```diff
@@ -14,51 +14,28 @@ _Collaborate and work together on GitHub._
...
 
-1. Right-click **Start course** and open the link in a new tab.
-2. In the new tab, most of the prompts will automatically fill in for you.
-   - For owner, choose your personal account or an organization to host the repository.
-   - We recommend creating a public repository, as private repositories will [use Actions minutes](https://docs.github.com/en/billing/managing-billing-for-github-actions/about-billing-for-github-actions).
-   - Scroll down and click the **Create repository** button at the bottom of the form.
-3. After your new repository is created, wait about 20 seconds, then refresh the page. Follow the step-by-step instructions in the new repository's README.
+1. Wait about 20 seconds then refresh this page (the one you're following instructions from). [GitHub Actions](https://docs.github.com/en/actions) will automatically update to the next step.
 
 <footer>
```
Comment (jusikXL) [inline README.md:40]: Nice footer

**Review by jusikXL — APPROVED** [review]

File: index.html (hunk 2 above)
Comment (jusikXL) [inline index.html:50]:
```suggestion
<h2 hidden>I suggest this</h2>
```

**Review by jusikXL — CHANGES_REQUESTED** [review]

File: index.html [hunk 4]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>I suggest this</h2>
```
Comment (jusikXL) [inline index.html:50]:
```suggestion
<h2 hidden>I suggest this bullshit</h2>
```

**Review by jusikXL — CHANGES_REQUESTED** [review]

File: index.html (hunk 4 above)
Comment (TheBestChelik) [reply index.html:50]: > bullshit

this is bullshit

**Review by jusikXL — APPROVED** [review]
Good job, my bad

**Pull request merged**